pywt/tests/test_swt.py

```diff
@@ -167,6 +169,7 @@ def test_swt2_ndim_error():
         assert_raises(ValueError, pywt.swt2, x, 'haar', level=1)
 
 
+@dec.slow
```

Comment:
Can we have one test not marked as slow for ``iswt2`` and ``iswtn``? The default for ``test()`` is not to run slow tests, so easy to be developing for some time without running any tests for these functions otherwise.